plugins/meta/portmap/portmap.go

```diff
@@ -225,19 +225,27 @@ func fillDnatRules(c *chain, config *PortMapConf, containerNet net.IPNet) {
 	c.rules = make([][]string, 0, 3*len(entries))
 	for _, entry := range entries {
 		// If a HostIP is given, only process the entry if host and container address families match
+		// and append it to the iptables rules
+		addRuleBaseDst := false
 		if entry.HostIP != "" {
 			hostIP := net.ParseIP(entry.HostIP)
 			isHostV6 := (hostIP.To4() == nil)
 
 			if isV6 != isHostV6 {
 				continue
 			}
+
+			// Unspecified addresses can not be used as destination
+			// TODO: check if we need to filter additional addresses
```

Comment:
We don't really want to merge a TODO - can you make a decision here?

Reply:
I think that is fine to filter only unspecified addresses, I was thinking about link-local or multicast but who knows 🤷‍♂️ , unspecified are the only ones that does not work for sure.